identity-service/src/routes/captcha.js

```diff
@@ -0,0 +1,37 @@
+const config = require('../config')
+const { handleResponse, successResponse, errorResponseForbidden, errorResponseBadRequest } = require('../apiHelpers')
+const { sequelize } = require('../models')
+
+module.exports = function (app) {
+  app.get('/scores', handleResponse(async (req, res, next) => {
+    if (req.headers['x-captcha-score'] !== config.get('captchaScoreSecret')) {
+      return errorResponseForbidden('Not permissioned to view captcha score.')
+    }
+
+    let response = {}
+    const queryParams = req.query
+
+    // If ids are not passed in, return
+    let userIds
+    if (queryParams && queryParams.id) {
+      userIds = queryParams.id
+    }
+    if (!userIds) return errorResponseBadRequest('No ids provided')
+
+    // If not valid ids are passed in, return
+    userIds = userIds.split(',').filter(id => parseInt(id))
+    if (!userIds.length) return errorResponseBadRequest('No valid ids provided')
+
+    const recaptchaEntry = await sequelize.query(
+      `select "Users"."blockchainUserId" as "userId", "Users"."walletAddress" as "wallet", "BotScores"."recaptchaScore" as "score"
```

Comment:
why do this join here? i think it should be fine to get user Id & scores

Reply:
I did this for this flow reason:
1. captcha score is calculated, add wallet | score | context + other fields, as wallet is passed in through the request body
2. to get the score, hit the GET route /users?id=1,2,3,..,..
3. connect ids to wallet addresses from data in the Users table
4. get the scores from wallet addresses from BotScores table

Reply:
discussed offline -- it's cus hedgehog does not pass in the userId! will keep this